UI/Store.cpp

```diff
@@ -35,7 +35,7 @@
 #include "UI/EmuScreen.h"
 #include "UI/Store.h"
 
-const std::string storeBaseUrl = "http://store.ppsspp.org/";
+const std::string storeBaseUrl = "https://store.ppsspp.org/";
```

Comment:
Since not all platforms support https://, we don't actually want to do this yet, right?

-[Unknown]

Reply:
True, fixing.